test.sh

```diff
@@ -40,7 +40,7 @@ for dir in ${VERSIONS}; do
   # Kept also IMAGE_NAME as some tests might still use that.
   IMAGE_NAME="$(docker inspect -f "{{.Config.Labels.name}}" "$IMAGE_ID"):$IMAGE_VERSION"
     # shellcheck disable=SC2268
-  if [ "${OS}" == "centos7" ] || [ "${OS}" == "c9s" ] || [ "${OS}" == "fedora" ]; then
+  if [ "${OS}" == "centos7" ] || [ "${OS}" == "c9s" ] || [ "${OS}" == "c10s" ] || [ "${OS}" == "fedora" ]; then
```

Comment:
What about to remove centos7?

Reply:
This will be removed by a different PR. It will remove RHEL7 and CentOS 7 functionality, as soon as our containers will stop support RHEL7